### PR TITLE
#1597 improving aria-label function by removing tab index on progress bar

### LINKED ISSUE #1597: Aria label on focusable div is read twice
Aria label on focusable div read twice. On rows with the spinner, it is read once per tab stop. On rows without the spinner, it is read twice, once for the spinner, and once for the focussable div.

MyLA Technical Review 7/24 - row 12

### PR DESCRIPTION
**Pull request by pushyamig**
Fixes #1597

### CODE REVIEW

**Comment by zqian** [comment]
@pushyamig is this still in draft mode?

**Comment by pushyamig** [comment]
>@pushyamig is this still in draft mode?

I was checking with Gonzalo, if the change fits the issue reported. I will open up after adding details of how to test it.

File: assets/src/components/ProgressBarV2.js [hunk 1]
```diff
@@ -50,7 +50,6 @@ function ProgressBarV2 (props) {
     <Root>
       {
         <div
-          tabIndex='0'
```
Comment (pushyamig) [inline assets/src/components/ProgressBarV2.js:53]: Removing the Tabindex is the suggestion from accessibility

**Review by zqian — APPROVED** [review]
The PR fulfilled two requested changes in #1597

**Pull request merged**